packages/frontend/ci/configFromEnvironment.js

```diff
@@ -0,0 +1,73 @@
+const assert = require("assert");
+
+const Environments = require("../../../features/environments.json");
+const { parseBooleanFromShell } = require("../src/config/envParsers");
+
+const Config = {
+  NEAR_WALLET_ENV: process.env.NEAR_WALLET_ENV,
+  TRAVIS: parseBooleanFromShell(process.env.TRAVIS),
```

Comment:
Could we put the above two keys in alphabetical order please? It makes it a lot easier to find values in bigger object hashes like this.